src/js/api/justification.js

```diff
@@ -0,0 +1,55 @@
+if (
+    typeof require !== "undefined" &&
+    (typeof window === "undefined" || typeof __webpack_require__ !== "undefined") // eslint-disable-line camelcase
+) {
+    // eslint-disable-next-line no-redeclare
+    var base = require("../base");
+    // eslint-disable-next-line no-redeclare
+    var ripe = base.ripe;
+}
+
+/**
+ * Gets the justifications list. Can be optionally filtered by context.
+ *
+ * @param {Object} options An object of options to configure the query and
+ * its results, such as:
+ * - 'filters[]' - List of filters that the query will use to, operators such as
+ * ('in', 'not_in', 'like', 'likei', 'llike', 'llikei', 'rlike', 'rlikei', 'contains'),
+ * (eg: 'context:eq:42') would filter by the 'context' that equals to '42'.

```

Comment:
Don't use context as an example, it gets a bit confusing given that it is a parameter.